.editorconfig

```diff
@@ -1,16 +1,16 @@
-root = true
-
-[*]
-indent_style = space
-indent_size = 4
-end_of_line = lf
-charset = utf-8
-insert_final_nweline = true
-max_line_length = 80
-trim_trailing_whitespace = true
-
-[*.md]
-trim_trailing_whitespace = false
-
-[COMMIT_EDITMSG]
-max_line_length = 70
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+insert_final_nweline = true
+max_line_length = 120
```

Comment:
The 80 limit length is largely set due to PEP, which I followed in my last job where I worked in Python. I'll think about whether or not we can overrule it here, but since it was used everywhere when writing this repository, I feel a bit weird about changing it now 

https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
Ah, didn't realize PEP covered that. No worries, it's just a little more cramped than I'm used to :)

Reply:
Yeah, I've been used to much longer line lenghts myself lately in my current job, going back is always a hassle, but I figured I made a styleguide decision back when I started it, should respect it for now :p